hivemq4/k8s-image/Dockerfile

```diff
@@ -0,0 +1,47 @@
+ARG BASE_IMAGE=hivemq/hivemq4:dns-latest
+ARG HIVEMQ_VERSION=4.7.5
+ARG PROMETHEUS_EXTENSION_VERSION=4.0.4
+
+FROM busybox as BUILD
+ARG HIVEMQ_VERSION
+ARG PROMETHEUS_EXTENSION_VERSION
+
+WORKDIR /opt/hivemq/extensions
+
+COPY hivemq-k8s-sync-extension-"${HIVEMQ_VERSION}".zip ./hivemq-k8s-sync-extension.zip
```

Comment:
Is it wanted for the k8s-sync-extension to be put into the folder before we can build the image?

Reply:
For the Jenkins build yes, as the gradle build the extension, also for tests it is better like that, so it can be build locally and the version set. Not sure if I answer your question correctly or I misunderstood.

Reply:
Yes it is